### PR TITLE
Add kokoro basics

### PR DESCRIPTION
**Pull request by akerekes**

### CODE REVIEW

**Comment by akerekes** [comment]
Integration tests for deploy (the ones that are run in the appveyor build) are not yet configured.

File: kokoro/ubuntu/presubmit.cfg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Location of the continuous build bash script in repository.
+build_file: "appengine-plugins-core/kokoro/continuous.sh"
```
Comment (patflynn) [inline kokoro/ubuntu/presubmit.cfg]: should this be using app-plugins-core from the maven plugin?

Comment (akerekes) [inline kokoro/ubuntu/presubmit.cfg]: No, this definitely should be `app-maven-plugin/kokoro/continuous.sh`

File: kokoro/windows/presubmit.cfg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Location of the continuous build bash script in repository.
+build_file: "appengine-plugins-core/kokoro/continuous.bat"
```
Comment (akerekes) [inline kokoro/windows/presubmit.cfg]: this should be `app-maven-plugin/kokoro/continuous.sh` as well

**Pull request merged**